app/components/polls/polls-edit/polls-edit-directive.js

```diff
@@ -54,18 +54,24 @@ angular.module('polls.edit.directive', [
         // in such a way that modal-on-close="validatePublication()"
         // fires before the scope model data has changed.
         $timeout(function () {
+          var pollForm = $scope.pollForm;
           var published = $scope.model.published;
           var endDate = $scope.model.end_date;
-          var publishedField = $scope.pollForm.published;
-          var endDateField = $scope.pollForm.endDate;
+          var publishedField = pollForm.published;
```

Comment:
It was an artifact of some debugging/changes to the way I was setting validations. Restored it.